mule-extensions-api/src/main/java/org/mule/runtime/extension/api/model/connection/ImmutableConnectionProviderModel.java

```diff
@@ -68,4 +72,12 @@ public ConnectionManagementType getConnectionManagementType() {
   public Set<ExternalLibraryModel> getExternalLibraryModels() {
     return externalLibraryModels;
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public StereotypeModel getStereotype() {
```

Comment:
inherit. is it fair to say that every component has a stereotype? Consider the things which are not executable components too